library/standard_tables_capture.inc

```diff
@@ -676,7 +676,36 @@ function valueset_import($type)
                         foreach ($nqf as $nqf_code) {
                             foreach ($cp->Concept as $con) {
                                 $con_attr = $con->attributes();
-                                sqlStatementNoLog("INSERT INTO valueset values(?,?,?,?,?,?,?) on DUPLICATE KEY UPDATE code_system = values(code_system),description = values(description),valueset_name = values(valueset_name)", array($nqf_code,$con_attr->code,$con_attr->codeSystem,$con_attr->codeSystemName,$vset_attr->ID,$con_attr->displayName,$vset_attr->displayName));
+                                sqlStatementNoLog(
+                                    "INSERT INTO valueset values(?,?,?,?,?,?,?) on DUPLICATE KEY UPDATE 
+                                    code_system = values(code_system),
+                                    description = values(description),
+                                    valueset_name = values(valueset_name)",
+                                    array(
+                                        $nqf_code,
+                                        $con_attr->code,
+                                        $con_attr->codeSystem,
+                                        $con_attr->codeSystemName,
+                                        $vset_attr->ID,
+                                        $con_attr->displayName,
+                                        $vset_attr->displayName
+                                    )
+                                );
+                                sqlStatementNoLog(
```

Comment:
If I'm following the logic correctly, this statement will have a lot of duplicates as the oid will be repeated many many times.  However, you've taken care of this with the ON DUPLICATE KEY UPDATE.  Correct?

Reply:
right, once again, copied the above code for `valueset`, thanks @adunsulag

Reply:
it's testing well, I'll work on trying to get `CMS69V10` to pass and report back if any issues related to this PR.